### ISSUE TITLE
🐞 Unable to access uploaded files while running fiber

### ISSUE THREAD

**Issue by hyingreborn** [gofiber/fiber#221]
**Fiber version/commit**
1.8.2

**Issue description**
Unable to access static folder uploaded files before restarting fiber

**Expected behavior**
I upload some static files to static folder，then unable to access these files, until restart the fiber server

**Steps to reproduce**

**Code snippet**

```go
package main

import "github.com/gofiber/fiber"

func main() {
  app := fiber.New()
  // set static path
  app.Static("/", "./public")
  // upload route
  app.Post("/api/v1/upload", func(c *fiber.Ctx){
    // ...
    c.SaveFile(file, "./public/upload/hashString.jpg")
  })
  // ...
}
```

**Comment by hyingreborn** [comment]
By the way, I'm also using minio for storage, and its method requires providing a path to the file. C.formfile () doesn't seem to have a temporary path to the file

**Comment by Fenny** [comment]
I tried to re-produce your issue:
```
create main.go
```
```go
package main

import (
	"fmt"

	"github.com/gofiber/fiber"
)

func main() {
	app := fiber.New()
	app.Post("/", func(c *fiber.Ctx) {
		if form, err := c.MultipartForm(); err == nil {
			files := form.File["documents"]
			for _, file := range files {
				fmt.Println(file.Filename, file.Size, file.Header["Content-Type"][0])
				// => "main.go" 546 "application/octet-stream"
				c.SaveFile(file, fmt.Sprintf("./copy_%s", file.Filename))
			}
		}
	})
	app.Listen(3000)
}

// curl -F documents=@main.go http://localhost:3000

```
Run:
```
curl -F documents=@main.go http://localhost:3000
```
Writing or accessing copy_main.go seems to work fine?

**Comment by hyingreborn** [comment]
```
package main

import (
  "fmt"

  "github.com/gofiber/fiber"
)

func main() {
  app := fiber.New()
  app.Static("/", "./public")
  app.Post("/", func(c *fiber.Ctx) {
    if form, err := c.MultipartForm(); err == nil {
      files := form.File["documents"]
      for _, file := range files {
        fmt.Println(file.Filename, file.Size, file.Header["Content-Type"][0])
        // => "main.go" 546 "application/octet-stream"
        c.SaveFile(file, fmt.Sprintf("./public/upload/copy_%s", file.Filename))
      }
    }
  })
  app.Listen(3000)
}
```
Then visit : http://localhost:3000/upload/filename.extname

Try uploading an image to the Static directory? @Fenny

**Comment by Fenny** [comment]
The following code works for me, could you try it?
Assuming you have a folder name `upload` inside the `public` directory.
```go
package main

import (
  "github.com/gofiber/fiber"
  "fmt"
  "io/ioutil"
)

func main() {
  app := fiber.New()
  app.Static("/", "./static")
  app.Post("/", func(c *fiber.Ctx) {
    if form, err := c.MultipartForm(); err == nil {
      files := form.File["documents"]
      for _, file := range files {
        fmt.Println(file.Filename, file.Size, file.Header["Content-Type"][0])
        c.SaveFile(file, fmt.Sprintf("./public/upload/copy_%s", file.Filename))
        dat, err := ioutil.ReadFile(fmt.Sprintf("./public/upload/copy_%s", file.Filename))
        if err != nil {
          panic(err)
        }
        fmt.Print(len(dat))
      }
    }
  })
  app.Listen(3000)
}
```
Curl commmand
```
curl -F documents=@main.go http://localhost:3000
```

**Comment by Fenny** [comment]
This should be fixed in `v1.8.32` https://github.com/gofiber/fiber/releases/tag/v1.8.32
Feel free to re-open this issue if you still have problems regarding this issue.

**Comment by hyingreborn** [comment]
Thanks! The problem was fixed
